compiler/aarch64/codegen/ARM64Debug.cpp

```diff
@@ -910,6 +919,32 @@ TR_Debug::print(TR::FILE *pOutFile, TR::ARM64Trg1Src1Instruction *instr)
    trfflush(_comp->getOutFile());
    }
 
+void
+TR_Debug::print(TR::FILE *pOutFile, TR::ARM64Trg1ZeroSrc1Instruction *instr)
+   {
+   printPrefix(pOutFile, instr);
+   TR::InstOpCode::Mnemonic op = instr->getOpCodeValue();
+
+   if (op == TR::InstOpCode::orrx || op == TR::InstOpCode::orrw)
+      {
+      // mov alias
+      trfprintf(pOutFile, "mov%c \t", (op == TR::InstOpCode::subsx) ? 'x' : 'w');
+      }
+   else if (op == TR::InstOpCode::subsx || op == TR::InstOpCode::subsw)
```

Comment:
`neg` is an alias of `sub` instead of `subs`.

Reply:
Fixed.